haskell-build/orb.yml

```diff
@@ -126,12 +133,13 @@ commands:
           name: Copying executable
           command: |
             mkdir -p ./build/dist
-            _os_suffix=$OS_NAME
-            if [ "$_os_suffix" = "darwin" ]; then _os_suffix="osx"; fi
-            # according to Cabal docs executables are located here
-            _exe_build_path="$PWD/dist-newstyle/build/$BUILD_ARCH-$BUILD_OS_NAME/ghc-$BUILD_GHC_VERSION/$BUILD_PROJECT_NAME-$BUILD_PROJECT_VERSION/x/"
-            # copy all the executables
-            find "$_exe_build_path" -type f -perm -a=x -regextype egrep -regex '^.+/x/(.+)/build/\1/\1$' -exec cp -- "{}" "$PWD/build/dist" \;
+
+            # Cabal symlinks its installs, so we need to create a place for it to symlink to, and then perform a hard copy.
+            mkdir ./tmp/
+            cabal v2-install --symlink-bindir=./tmp/
+            cp -R -L ./tmp/* ./build/dist/
```

Comment:
please don't use `/tmp/` this. You can use `/tmp/blah-blah`, but `/tmp` may (and will) contain other stuff that we don't want to be copied with the binaries.

Reply:
That makes sense. Will fix.